src/app/(app)/missing-shoes/add/components/MissingShoesFormBuilder.tsx

```diff
@@ -99,90 +101,103 @@ export const MissingShoesFormBuilder = () => {
-                <FormItem>
-                  <FormLabel>Size</FormLabel>
-                  <FormControl>
-                    <Input placeholder="e.g., 42, 8.5, 9" {...field} />
-                  </FormControl>
-                  <FormMessage />
-                </FormItem>
-              )}
-            />
-            <FormField
-              control={form.control}
-              name={"time"}
-              render={({ field }) => (
-                <FormItem>
-                  <FormLabel>Time Lost</FormLabel>
-                  <FormControl>
-                    <DateTimePicker
-                      time={field.value}
-                      onChangeTime={field.onChange}
-                    />
-                  </FormControl>
-                  <FormMessage />
-                </FormItem>
-              )}
-            />
+          <FormField
+            control={form.control}
+            name={"color"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Color</FormLabel>
+                <FormControl>
+                  <Input placeholder="e.g., Red, Blue, Black" {...field} />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+          <FormField
+            control={form.control}
+            name="type"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Shoes Type</FormLabel>
+                <FormControl>
+                  <Select
+                    {...field}
+                    value={field.value}
+                    onValueChange={field.onChange}
+                  >
+                    <SelectTrigger data-testid="leave-type" id="type">
+                      <SelectValue placeholder="Select shoe type" />
+                    </SelectTrigger>
+                    <SelectContent>
+                      {SHOES_TYPES.map((type) => (
+                        <SelectItem key={type} value={type}>
+                          {type}
+                        </SelectItem>
+                      ))}
+                    </SelectContent>
+                  </Select>
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+          <FormField
+            control={form.control}
+            name={"size"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Size</FormLabel>
+                <FormControl>
+                  <Input placeholder="e.g., 42, 8.5, 9" {...field} />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+          <FormField
+            control={form.control}
+            name={"time"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Time Lost</FormLabel>
+                <FormControl>
+                  <DateTimePicker
```

Comment:
Can we make sure this is date-time picker. Having capabilities to pick both date and exact time?

Reply:
Yes sir this date-time picker will pick exact date and time